ogi/server/utils/evm.ts

```diff
@@ -0,0 +1,21 @@
+import { createThirdwebClient } from 'thirdweb'
+import {
+  base,
+  optimism,
+  avalanche,
+  polygon,
+  type ChainOptions,
+} from 'thirdweb/chains'
+
+export const chains: {
+  [key: string]: Readonly<ChainOptions & { rpc: string }>
+} = {
+  base,
+  optimism,
+  avalanche,
+  polygon,
```

Comment:
imx?

Reply:
lets follow Prefix naming instead in here